tools/rna_tools/rnacode/rnacode.xml

```diff
@@ -1,7 +1,8 @@
-<tool id="rbc_rnacode" name="RNAcode" version="0.3.1">
+<tool id="rbc_rnacode" name="RNAcode" version="0.3+galaxy2">
```

Comment:
@bernt-matthias last coment. I think we need to keep it with `0.3.2` as `0.3.1` is greater then `0.3+galaxy2` afaik.

Reply:
indeed. changed it